Unifi - 404 when installing mongo 4.4

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

UniFi Network Server

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/unifi.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

When installing Unifi on a non-AVX cpu, the script tries to install mongo 4.4 (the last version that doesnt require AVX) but fails when trying to download a specific version of libssl that no longer exists under that url.

### 🔄 Steps to reproduce the issue.

1. Run this install script on a non-AVX cpu.

### ❌ Paste the full error output (if available).

```  
  ✔️   No AVX Support Detected
  ⏳   Installing MongoDB 4.4curl: (22) The requested URL returned error: 404

[ERROR] in line 31: exit code 0: while executing command curl -fsSL "http://security.debian.org/debian-security/pool/updates/main/o/openssl/ " -o "libssl1.1_1.1.1n-0+deb10u6_amd64.deb"

[ERROR] in line 1353: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/${var_install}.sh)"
```

### 🖼️ Additional context (optional).

_No response_

--- a/install/unifi-install.sh
+++ b/install/unifi-install.sh
@@ -28,8 +28,8 @@ if ! grep -q -m1 'avx[^ ]*' /proc/cpuinfo; then
   msg_ok "No AVX Support Detected"
   msg_info "Installing MongoDB 4.4"
   if ! dpkg -l | grep -q "libssl1.1"; then
-    curl -fsSL "http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1n-0+deb10u6_amd64.deb" -o "libssl1.1_1.1.1n-0+deb10u6_amd64.deb"
-    $STD dpkg -i libssl1.1_1.1.1n-0+deb10u6_amd64.deb
+    curl -fsSL "https://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1w-0+deb11u3_amd64.deb" -o "libssl1.1_1.1.1w-0+deb11u3_amd64.deb"
+    $STD dpkg -i libssl1.1_1.1.1w-0+deb11u3_amd64.deb
   fi
   curl -fsSL "https://www.mongodb.org/static/pgp/server-4.4.asc" | gpg --dearmor >/usr/share/keyrings/mongodb-server-4.4.gpg
   echo "deb [signed-by=/usr/share/keyrings/mongodb-server-4.4.gpg] https://repo.mongodb.org/apt/debian buster/mongodb-org/4.4 main" >/etc/apt/sources.list.d/mongodb-org-4.4.list

